.test/config-simple/config.yaml

```
@@ -12,7 +12,10 @@ ref:
   release: 100
   # Genome build
   build: R64-1-1
-
+  pangenome:
+    activate: false
+    index: ""
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)